contracts/multisig/contracts/src/MultiSigWalletWithTimeLock.sol

```diff
@@ -18,15 +18,18 @@
 
 pragma solidity ^0.5.9;
 
+import "@0x/contracts-utils/contracts/src/SafeMath.sol";
 import "./MultiSigWallet.sol";
 
 
 /// @title Multisignature wallet with time lock- Allows multiple parties to execute a transaction after a time lock has passed.
 /// @author Amir Bandeali - <amir@0xProject.com>
 // solhint-disable not-rely-on-time
 contract MultiSigWalletWithTimeLock is
-    MultiSigWallet
+    MultiSigWallet,
+    SafeMath
```

Comment:
I'd prefer `using SafeMath for uint256`

Reply:
We can't do that in this case since `SafeMath` is a contract instead of a library. I'm up for changing this to a library (`SafeMath` is typically a library); however, it may affect a lot of code.